### ISSUE TITLE
[$10][MSFT-186] [Challenge Feed][NVD] Facebook icon is not properly read-out by the screen reader

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2854]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges/30095347/?type=develop&nocache=true
2. Hover over the Facebook icon

**Expected Result**
It should be "Icon Facebook"

**Actual Result**
Read-out as "f"

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F65

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096063 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096063 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096063 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by veshu** [comment]
@r0hit-gupta Please use `changes for #xxxx` style of commit message.

**Comment by veshu** [comment]
@r0hit-gupta Also please use `qa-accessibility` branch.

**Comment by r0hit-gupta** [comment]
@veshu the branch `qa-accessibility` has lint errors for the latest commit. This would fail every PR if not fixed. How should I proceed?

**Comment by veshu** [comment]
@r0hit-gupta Please fix the https://github.com/topcoder-platform/community-app/pull/2914

**Comment by r0hit-gupta** [comment]
@veshu done.

**Comment by gurmeetb** [comment]
@r0hit-gupta we will need a new PR for this, using `qa-accessibility` as base. I changed the base on PR and see huge lot of changes.

**Comment by r0hit-gupta** [comment]
@gurmeetb sure. Working on it

**Comment by r0hit-gupta** [comment]
New PR #2964

**Comment by anonymousjaggu** [comment]
@r0hit-gupta please check the comment in MR

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2854 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2854 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097099 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097099 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by anonymousjaggu** [comment]
Duplicate with #2825

**Comment by tosha5252** [comment]
Verified working

Env: 
Windows 10
Browser: Chrome

**Comment by sushilshinde** [comment]
@r0hit-gupta which PR did fix this issue?

**Comment by r0hit-gupta** [comment]
@sushilshinde it was https://github.com/topcoder-platform/community-app/pull/3057

**Comment by tosha5252** [comment]
This is not working in the production environment in Chrome browser.  None of the links in the footer are being read out.

**Comment by crazyk07** [comment]
@sushilshinde  Same thing here.  Not working in Production.  More context:

When tabbing through the icons, they are read out properly, but when hovering with the mouse on the FB Icon, it just says "F".  

Using NVDA.

**Comment by tosha5252** [comment]
@sushilshinde still getting the same thing, not working for me in Chrome

**Comment by sushilshinde** [comment]
This is the case where code is in the production

This was the PR that was approved 
https://github.com/topcoder-platform/community-app/pull/3057/files

And same code is in the production 
https://github.com/topcoder-platform/community-app/blob/master/src/shared/components/challenge-detail/Specification/SideBar/ShareSocial.jsx

also here is the screenshot 

![Sample Challenge for Test for Accessibility Challenge 2019-07-30 13-07-22](https://user-images.githubusercontent.com/683231/62110059-32173e00-b2cb-11e9-8016-0c52b59d5b6b.png)

**Comment by lakshmiathreya** [comment]
This is working on Chrome/Prod url - could you double check @tosha5252

**Comment by lakshmiathreya** [comment]
fyi @nkumar-topcoder @sushilshinde

**Comment by drasticdpk** [comment]
@lakshmiathreya  still I am having issue. Please confirm. 
https://monosnap.com/file/PQpBGL8XJrECAs9WvwgXgT9P9zyT45

**Comment by lakshmiathreya** [comment]
@drasticdpk - you are right - Chrome on Mac reads out Facebook correctly on Voice Over. Chrome on Window 10 Jaws reads it out as F.

**Comment by r0hit-gupta** [comment]
@lakshmiathreya @drasticdpk @sushilshinde it might be due to the fact that the facebook is not exactly an icon image but a `div`.

https://github.com/topcoder-platform/community-app/blob/a56a0f8ccc5b047eab986609aafd6dbd3818e9f4/src/shared/components/challenge-detail/Specification/SideBar/ShareSocial.jsx#L43

**Comment by r0hit-gupta** [comment]
New PR at https://github.com/topcoder-platform/community-app/pull/3164 

Should solve this issue for once and all. Please review.

**Comment by lakshmiathreya** [comment]
Verified hover on Facebook icon reading out 'Facebook'... Win10/Chrome/NVDA & MacOS/Chrome/VoiceOver ... Closing Issue ...

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097099<br/><br/>```This is an automated message for crazyk via Topcoder X```
